O/Octave/build_tarballs.jl

```diff
@@ -0,0 +1,60 @@
+using BinaryBuilder, Pkg
+
+name = "Octave"
+version = v"9.2.0"
+
+# Collection of sources required to build Octave
+sources = [
+   ArchiveSource("https://ftpmirror.gnu.org/octave/octave-9.2.0.tar.gz",	
+                  "0636554b05996997e431caad4422c00386d2d7c68900472700fecf5ffeb7c991"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/octave*
+
+export CPPFLAGS="-I${includedir}"
+export LDFLAGS="-L${libdir}"
+
```

Comment:
Are these necessary?  I think they may be already in the compiler search paths.

Reply:
Just copied from another file somewhere to get started. I was having trouble with BLAS and LAPACK libraries which it wasn't finding. Let me remove and try again.